packages/image_picker/image_picker_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.7+6
+
+* Fixes NullPointerException on Android 10 devices when using Android Photo Picker.
```

Comment:
This note and another comment later indicate that this PR is targeted at fixing this issue for Android 10. On the other hand the linked issue (https://github.com/flutter/flutter/issues/104168) seems to occur on a range of versions (with the initial report indicating Android 12). 

If you are linking it as fixed (and therefore saying that this fix isn't specific to Android 10) then the CHANGELOG and comments here should probably not indicate that, correct?

Reply:
Yes the issue does seem to happen in several pre-Android 13 devices, but in my app's case, most of the crashes are from Android 10 devices. but will change the changelog note in the next commit